packages/Utils/test/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Miscellaneous_test
+  SOURCES tstMiscellaneous.cpp ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
```

Comment:
OK - I wonder why it is not running if comm is removed then. We really want all of our tests to execute in both modes usually so we could probably just remove this